pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/OpplysningerBruktIBeregningenAlder.kt

```diff
@@ -0,0 +1,394 @@
+                )
+            }
+
+            showIf(regelverkstype.isOneOf(AP2016)) {
+                paragraph {
+                    text(
+                        Bokmal to "For den delen av pensjonen din som er beregnet etter regler i kapittel 19 har vi brukt disse opplysningene i beregningen vår:",
+                        Nynorsk to "For den delen av pensjonen din som er berekna etter reglar i kapittel 19, har vi brukt desse opplysningane i berekninga vår:",
+                        English to "We have used the following information to calculate the part of your pension that comes under the provisions of Chapter 19:",
+                    )
+                }
+            }
+
+            includePhrase(
+                OpplysningerBruktIBeregingTabellKap19(
+                    trygdetidsdetaljerKap19VedVirk = trygdetidsdetaljerKap19VedVirk,
+                    tilleggspensjonVedVirk = tilleggspensjonVedVirk,
+                    beregnetPensjonPerManedVedVirk = beregnetPensjonPerManedVedVirk,
+                    beregningKap19VedVirk = beregningKap19VedVirk,
+                    beregningKap20VedVirk = beregningKap20VedVirk,
+                    yrkesskadeDetaljerVedVirk = yrkesskadeDetaljerVedVirk,
+                    alderspensjonVedVirk = alderspensjonVedVirk,
+                )
+            )
+
+            showIf(regelverkstype.isOneOf(AP2016)) {
+                includePhrase(
+                    OpplysningerBruktIBeregningTabellKap20(
+                        beregnetPensjonPerManedVedVirk = beregnetPensjonPerManedVedVirk,
+                        beregningKap19VedVirk = beregningKap19VedVirk,
+                        trygdetidsdetaljerKap20VedVirk = trygdetidsdetaljerKap20VedVirk,
+                        beregningKap20VedVirk = beregningKap20VedVirk,
+                        krav = krav
+                    )
+                )
+            }
+
+            includePhrase(
+                OpplysningerBruktIBeregningenTrygdetidTabeller(
+                    trygdetidsdetaljerKap19VedVirk = trygdetidsdetaljerKap19VedVirk,
+                    trygdetidsdetaljerKap20VedVirk = trygdetidsdetaljerKap20VedVirk,
+                    beregningKap19VedVirk = beregningKap19VedVirk,
+                    beregningKap20VedVirk = beregningKap20VedVirk,
+                    alderspensjonVedVirk = alderspensjonVedVirk,
+                    inngangOgEksportVurdering = inngangOgEksportVurdering,
+                    trygdetidNorge = trygdetidNorge,
+                    trygdetidEOS = trygdetidEOS,
+                    trygdetidAvtaleland = trygdetidAvtaleland,
+                )
+            )
+
+            showIf(
+                (regelverkstype.isOneOf(AP2016) and poengrekkeVedVirk.pensjonspoeng_safe.ifNull(emptyList()).size()
+                    .greaterThan(0))
+                        or (regelverkstype.isOneOf(AlderspensjonRegelverkstype.AP2011)
```

Comment:
Burde vi her også sjekke metode=="FOLKETRYGD"? (ref Script_vedleggBeregnIngenPensjonsOpptjening_001)

Reply:
_Men_ eg trur blokka `Du er registrert med ingen eller for lav inntekt til å ha rett til tilleggspensjon. For å ha rett til tilleggspensjon må du ha minst tre år med pensjonspoeng. Du får pensjonspoeng for år med inntekt over folketrygdens grunnbeløp (G) eller omsorgspoeng.` frå samme frase manglar.